### PR TITLE
Updated InputLabel.styles.ts, FormTextInput.tsx, and LoginScreen.tsx

### PR DESCRIPTION
**Pull request by CMChemist**
Changes I made:
- Updated InputLabel.styles.ts to match current UX/UI design.
- Added a placeholder prop to FormTextInput.tsx and updated DonorRegistrationScreen.tsx and ClientRegistrationScreen.tsx so that all calls to the FormTextInput component also passes an empty string to placeholder prop.
- Added placeholders to the TextInput components on the LoginScreen.tsx.

### CODE REVIEW

File: src/elements/FormTextInput/FormTextInput.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import styles from './FormTextInput.styles';
 
 interface FormTextInputProps {
 	text: string;
+	placeholder: string;
```
Comment (jacksonrya) [inline src/elements/FormTextInput/FormTextInput.tsx]: Add a question mark at the end of the property name, like 'width'. This makes the property optional within the interface

File: src/elements/FormTextInput/FormTextInput.tsx [hunk 2]
```diff
@@ -22,6 +23,7 @@ interface FormTextInputProps {
 
 export default ({
 	text,
+	placeholder,
```
Comment (jacksonrya) [inline src/elements/FormTextInput/FormTextInput.tsx]: `placeholder = '',` will set the default value of placeholder to an empty string. Now you can delete every instance where you passed an empty string placeholder into the component

File: src/elements/FormTextInput/InputLabel/InputLabel.styles.ts [hunk 3]
```diff
@@ -7,7 +7,10 @@ export default StyleSheet.create({
 	},
 	text: {
 		fontFamily: 'open-sans-regular',
```
Comment (jacksonrya) [inline src/elements/FormTextInput/InputLabel/InputLabel.styles.ts]: open-sans-bold

File: src/elements/FormTextInput/InputLabel/InputLabel.styles.ts [hunk 4]
```diff
@@ -7,7 +7,10 @@ export default StyleSheet.create({
...
-		fontSize: 14,
+		fontStyle: 'normal',
+		fontWeight: 'bold',
```
Comment (jacksonrya) [inline src/elements/FormTextInput/InputLabel/InputLabel.styles.ts]: no fontWeight property needed

File: src/screens/LoginScreen/LoginScreen.tsx [hunk 5]
```diff
@@ -55,6 +55,7 @@ export default () => {
 				autoCorrect={false}
 				autoFocus={true}
 				blurOnSubmit={false}
+				placeholder = 'example@thebgp.com'
```
Comment (jacksonrya) [inline src/screens/LoginScreen/LoginScreen.tsx]: Make sure to set up eslint auto-formating in your IDE so that these extra spaces around the equal signs are removed automatically. Same for line 71

File: src/elements/FormTextInput/FormTextInput.tsx [hunk 6]
```diff
@@ -43,6 +45,7 @@ export default ({
 						style={[ styles.input, disabled && styles.disabled, style, { textAlign: 'right' } ]}
 						autoCapitalize={autoCapitalize}
 						editable={!disabled}
+						placeholder={placeholder}
 					/>
```
Comment (jacksonrya) [inline src/elements/FormTextInput/FormTextInput.tsx]: Add placeholderTextColor prop set to global navy blue

File: src/elements/FormTextInput/FormTextInput.tsx [hunk 7]
```diff
@@ -56,6 +59,7 @@ export default ({
 					style={[ disabled && styles.disabled, styles.input, style ]}
 					autoCapitalize={autoCapitalize}
 					editable={!disabled}
+					placeholder={placeholder}
 				/>
```
Comment (jacksonrya) [inline src/elements/FormTextInput/FormTextInput.tsx]: Add placeholderTextColor prop set to global navy blue

File: src/screens/LoginScreen/LoginScreen.styles.ts [hunk 8]
```diff
@@ -11,8 +11,9 @@ export default StyleSheet.create({
 		height: 50,
 		marginBottom: 15,
 		backgroundColor: 'white',
```
Comment (jacksonrya) [inline src/screens/LoginScreen/LoginScreen.styles.ts]: Revert this file

The PR changes should be constrained to the [Trello card](https://trello.com/c/1l1Q7j24). So no need to update the login screen. Another PR request will handle the unique constraints of the login form fields.

File: src/screens/LoginScreen/LoginScreen.tsx [hunk 9]
```diff
@@ -55,6 +55,7 @@ export default () => {
 				autoCorrect={false}
```
Comment (jacksonrya) [inline src/screens/LoginScreen/LoginScreen.tsx]: Revert this file

The PR changes should be constrained to the [Trello card](https://trello.com/c/1l1Q7j24). So no need to update the login screen. Another PR request will handle the unique constraints of the login form fields.

File: src/util/colors.ts [hunk 10]
```diff
@@ -9,4 +9,4 @@ export const LIGHT_GRAY = '#F0EEEE';
 export const LIGHT_GRAY_DISABLED = '#C4C4C4';
 export const DARK_GRAY = 'rgb(60, 60, 60)';
 export const DARK_GRAY_TRANSPARENT = 'rgba(60, 60, 60, .5)';
-export const RED = '#E10000';
+export const RED = '#E10000';
```
Comment (jacksonrya) [inline src/util/colors.ts]: This file is out of scope of the card. Is this change a remnant of the commit where you updated colors yourself before you merged to master?

Comment (CMChemist) [reply src/util/colors.ts]: Yes. Every time I pulled from the upstream, my forked version of master would update to the correct colors.ts file. But when  I rebased my branched with my master, the old colors.ts file in my branch would overwrite the up to date color.ts file in master.

So, I had to copy and paste the correct version of colors.ts into my branch.

 I also thought I had to submit a new pull request once I finished the changes on this.

That PR is #36. How do we resolve that?

Comment (jacksonrya) [reply src/util/colors.ts]: Alright! We can go through it over the vid chat :)

And you're right about the new pull request, I'll close this one now!
